tool/ruby_vm/views/_trace_instruction.erb

```diff
@@ -11,6 +11,13 @@
 INSN_ENTRY(<%= insn.name %>)
 {
     vm_trace(ec, GET_CFP());
-    DISPATCH_ORIGINAL_INSN(<%= insn.jump_destination %>);
+    <%-
+    if insn.name =~
+    /\Atrace_opt_(plus|minus|mult|div|mod|eq|neq|lt|le|gt|ge|ltlt|and|or|aref|aset|length|size|empty_p|nil_p|succ|not|regexpmatch2)\z/
+      jump_dest = "opt_send_without_block"
+    end
+    -%>
+    <%= 'ADD_PC(1);' if insn.name == 'trace_opt_neq' %>
```

Comment:
You could calculate `1` instead of hard-coding it if you look at insn attributes, but I guess you thought this is easier to understand what's going on?

Reply:
Yes, I found it easier to just hard code it, since it was only needed in this one case.